### PR TITLE
[Keyboard] add dude09

### PR DESCRIPTION
**Pull request by astarryknight**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/starrykeebs/dude09/info.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "matrix_pins": {
+        "cols": ["C6", "D7", "E6"],
+        "rows": ["D1", "D0", "D4"]
+    },
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x534B"
+    },
```
Comment (dunk2k) [inline keyboards/handwired/starrykeebs/dude09/info.json:24]: Add Community Layout support
```suggestion
    },
    "community_layouts": ["ortho_3x3"],
```

File: keyboards/handwired/starrykeebs/dude09/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (dunk2k) [inline keyboards/handwired/starrykeebs/dude09/keymaps/via/rules.mk:1]:
```suggestion
VIA_ENABLE = yes

```

File: keyboards/handwired/starrykeebs/dude09/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# dude09
+
+![dude09](https://i.imgur.com/sBR8L7x.jpg)
```
Comment (dunk2k) [inline keyboards/handwired/starrykeebs/dude09/readme.md:3]:
```suggestion
![dude09](https://i.imgur.com/sBR8L7xh.jpg)
```

File: keyboards/handwired/starrykeebs/dude09/rules.mk [hunk 4]
```diff
@@ -0,0 +1 @@
+# This file intentionally left blank
```
Comment (dunk2k) [inline keyboards/handwired/starrykeebs/dude09/rules.mk:1]:
```suggestion
# This file intentionally left blank

```

File: keyboards/handwired/starrykeebs/dude09/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# dude09
```
Comment (dunk2k) [inline keyboards/handwired/starrykeebs/dude09/readme.md:1]:
```suggestion
# Dude09
```

File: keyboards/handwired/starrykeebs/dude09/info.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    "maintainer": "StarryKnight",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
```
Comment (dunk2k) [inline keyboards/handwired/starrykeebs/dude09/info.json:13]:
```suggestion
        "mousekey": true
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/handwired/starrykeebs/dude09/keymaps/via/rules.mk (hunk 2 above)
Comment (dunk2k) [reply keyboards/handwired/starrykeebs/dude09/keymaps/via/rules.mk]: Blank new line yes 🙂

File: keyboards/handwired/starrykeebs/dude09/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+A simple 3x3 macropad based on the VOID9 case.
+
+* Keyboard Maintainer: [astarryknight](https://github.com/astarryknight)
+* Hardware Supported: rp2040 (using the adafruit kb2040), firmware written for ATmega32U4
+* Hardware Availability: Amazon (https://amazon.com) & Digikey (https://https://www.digikey.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/starrykeebs/dude09:default  CHECK THIS
```
Comment (waffle87) [inline keyboards/handwired/starrykeebs/dude09/readme.md:13]:
```suggestion
    make handwired/starrykeebs/dude09:default
```

**Review by waffle87 — APPROVED** [review]

**Pull request merged**